### PR TITLE
Blog-Writer Feature

### PR DESCRIPTION
**Pull request by artificialfintelligence**
# Description
> Describe your pull request here. What does this change implement or fix? Include motivation and context, e.g. relevant logs, error output, etc.
First iteration of blog-writer AKA blogger (Work in Progress; this is the version I used in the May 3rd demo)

> Add directions for contributors. For example, do developers need to run `poetry install` after picking up these changes?
I will write a proper how-to guide, but for now:
In addition to the instructions provided in the "PDF-Reader" demo, run:
``` bash
pip install -r requirements.txt
```

# Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [X] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Maintenance
- [ ] New release

# Related issues
> Mention related GitHub and Linear issues. E.g. Closes #xxx or Fixes #xxx. Otherwise delete this section.
Issue #285 

# Checklists
To speed up the review process, please follow these checklists:

## Development
- [X] The Pull Request is small and focused on one topic
- [X] Lint rules pass locally (`make format && make lint`)
- [X] The code changed/added as part of this pull request has been covered with tests
- [X] All tests related to the changed code pass in development (`make test`)
- [X] The changes generate no new warnings (or explain any new warnings and why they're ok)
- [X] Commit messages are detailed
- [X] Changed code is self-explanatory and/or I added comments
- [X] I updated the documentation (docstrings, /docs)
See the testing guidelines for help on tests, especially those involving web services.

## Code review
- [X] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached.
- [X] I have performed a self-review of my code
- [ ] Issue from task tracker has a link to this pull request

💔 Thank you for submitting a pull request!

### CODE REVIEW

**Comment by 20001LastOrder** [comment]
@artificialfintelligence Could you add a readme briefly documenting the purpose and use of the blog writer?

**Comment by artificialfintelligence** [comment]
@20001LastOrder Done! Please review.

**Comment by 20001LastOrder** [comment]
Thanks for adding the tutorial @artificialfintelligence . Could you also add a `README.md` file and put the link to the tutorial in it?

**Comment by artificialfintelligence** [comment]
@20001LastOrder And done. Please note the link should work once the PR is merged. I wasn't sure if it would be automatically updated if I referenced the `blogger` branch.

File: demo/blog_writer/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+pdfminer.six
```
Comment (20001LastOrder) [inline demo/blog_writer/requirements.txt:1]: I think you forgot sherpa-ai as a dependency.? 😄

**Review by 20001LastOrder — APPROVED** [review]
LGTM

**Pull request merged**